src/views/HomePage/HomePage.tsx

```diff
@@ -2,19 +2,42 @@ import { TitleBanner } from '@ui/components/TitleBanner';
 import StyledHomePage from './StyledHomePage';
 import logo from '/wwtbtfc.svg';
 import { NavigationButton } from '@ui/components/NavigationButton';
+import colors from '@ui/styles/colors';
 
 /**
  * Main home page for the app
  */
 export const HomePage = () => {
+  const oneLiners = [
+    'because affection is a zero sum game',
+    "because there's not enough tension in this family",
+    'because it was getting too chummy around here',
+    'because mama raised exactly one loser',
+    "because there's not enough room in this house for the both of us",
+  ];
   return (
     <>
       <StyledHomePage>
         <TitleBanner title="" />
         <div>
           <img src={logo} className="logo" alt="WWTBTFC logo" />
         </div>
-        <h1>Who Wants to be the Favorite Child?</h1>
+        <h1
+          style={{
```

Comment:
Let's move these style objects into the `<StyledHomePage>` component. Add a `h1` and `h3` css selector to the body of the StyledHomePage component